src/lib/stringifier.js

```diff
@@ -73,7 +73,8 @@ module.exports = function (calc, node, originalValue, options, result, item) {
   let str = stringify(node, options.precision);
 
   const shouldPrintCalc =
-    node.type === 'MathExpression' || node.type === 'Function';
+    node.type === 'MathExpression' || node.type === 'Function' ||
+    node.type === 'ParenthesizedExpression';
 
   if (shouldPrintCalc) {
     // if calc expression couldn't be resolved to a single value, re-wrap it as
```

Comment:
Maybe you can remove the double round brackets by changing line 81-82:
```javascript
if (node.type === 'ParenthesizedExpression') {
 str = `${calc}${str}`;
else {
  str = `${calc}(${str})`;
}

Reply:
Good call, that gets the job done and removes any extra parentheses whether it's one or many. Will update this now.